doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,190 @@ Table 7-2 shows security capabilities
     tests)
+
+<a name="7.11"></a>
+## 7.11 Consolidated Security Requirements
+
+Please note that **Virtual Resource Manager (VRM)** is used to collectively represent VIM, K8s, …
+
+<a name="7.11.1"></a>
+### 7.11.1. System Hardening
+
+|  Ref | Requirement  |  Unit  | Definition  |
+|-------|------|------|-------|
+| sec.gen.001 | The Platform **must** maintain the state to what it is specified to be and does not change unless through change management process. |   |  |
+| sec.gen.002 | All systems part of NFVI **must** support password hardening (strength and rules for updates (process), storage and transmission, etc.) |  | Hardening: NIST SP 800-63B |
+| sec.gen.003 | All servers part of NFVI **must** support secure boot, root of trust and trusted boot |  | Secure boot - measured boot, verified boot  <br/> Root of trust: specified in RA or RI; should CNTT specify how? |
+| sec.gen.004 | The Operating Systems of all the servers part of NFVI **must** be hardened |  | NIST SP 800-123 |
+| sec.gen.005 | The Platform **must** support Operating System level access control |  | Details on OS |
+| sec.gen.006 | The Platform **must** support Secure logging |  | Details |
+| sec.gen.007 | All servers part of NFVI **must** be Time synchronized with authenticated Time service |  | |
+| sec.gen.008 | All servers part of NFVI **must** be regularly updated to address security vulnerabilities |  | |
+| sec.gen.009 | The Platform **must** support Software integrity protection and verification |  | |
+| sec.gen.010 | The NFVI **must** support Secure storage (all types) |  | Expand/Delete based on other reqts |
+| sec.gen.011 | The NFVI **should** support Read and Write only storage partitions (write only permission to one or more authorized actors) |  | NIST reference? |
+| sec.gen.012 | The Platform **must** ensure that only authorized actors have physical access to the underlying infrastructure. |  | |
+| sec.gen.013 | The Platform **must** ensure that only authorized actors have logical access to the underlying infrastructure. |  | |
+
+<a name="7.11.21"></a>
+###  7.11.2. Platform Access
+
+Ref | Requirement | Unit | Definition |
+|-------|-------|-------|---------|
+| sec.sys.001 | The Platform **must** support authenticated and secure APIs, API endpoints |  | |
+| | The Platform **must** implement authenticated and secure access to GUI |  | |
+| sec.sys.002 | The Platform **must** support Traffic Filtering for workloads (for example, Fire Wall) |  | |
+| sec.sys.003 | The Platform **must** support Secure and encrypted communications, and confidentiality and integrity of network traffic |  | |
+| sec.sys.004 | The NFVI **must** support Secure network channels |  | A secure channel enables transferring of data that is resistant to overhearing and tampering |
+| sec.sys.005 | The NFVI **must** segregate the underlay and overlay networks |  | |
+| sec.sys.006 | The NFVI must be able to utilize the VRM identity management capabilities |  | |
```

Comment:
How can NFVI utilise VRM (ex VIM) identity management capabilities?

Reply:
@msidana maybe needs an explanation but the NFVI includes the VMs (and hence the workloads) and they should be able to utilise the appropriate Identity service APIs.

Reply:
Hmm. VM's in this context is very generic. We should do some re-phrasing if we mean workloads - ```Workloads may be able to utilise VRM identity capabilities depending on the need of running application```.
++ I would also propose ```may``` here

Reply:
@msidana is this change considered optional or mandatory?

Reply:
Optional, according to me.